webapi/webui.py

```diff
@@ -67,7 +67,9 @@ def add_torrent(self, metainfo, options=None):
             options = {}
 
         coreconfig = component.get('CoreConfig')
-        options.update({'download_location': coreconfig.get("download_location")})
+
+        if 'download_location' not in options.keys() or options.get('download_location') == 'download_location':
```

Comment:
Thank you. I have no access to deluge now, so I'd like to ask to to recheck one thing.
This assumption seems strange: `options['download_location'] == 'download_location'`. Is it indeed how it should be?